File: task-03/HelloWorld.js
function isPrime(num) {
    if (num < 2) return false;
    for (let i = 2; i <= Math.sqrt(num); i++) {
        if (num % i === 0) return false;
    }
    return true;
}

function findPrimes(n) {
    const primes = [];
    for (let num = 2; num <= n; num++) {
        if (isPrime(num)) primes.push(num);
    }
    return primes;
}

const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('Enter a number: ', (n) => {
  console.log(`Prime numbers up to ${n} are: ${findPrimes(parseInt(n)).join(', ')}`);
  rl.close();
});

File: task-03/HelloWorld.go
package main

import (
    "fmt"
    "math"
)

func isPrime(num int) bool {
    if num < 2 {
        return false
    }
    for i := 2; i <= int(math.Sqrt(float64(num))); i++ {
        if num%i == 0 {
            return false
        }
    }
    return true
}

func findPrimes(n int) []int {
    primes := []int{}
    for num := 2; num <= n; num++ {
        if isPrime(num) {
            primes = append(primes, num)
        }
    }
    return primes
}

func main() {
    var n int
    fmt.Print("Enter a number: ")
    fmt.Scanln(&n)
    fmt.Printf("Prime numbers up to %d are: %v\n", n, findPrimes(n))
}
